book/analysis/introduction.md

```diff
@@ -146,5 +143,5 @@ including the programming language, compiler, OS, machine, and time of day.
 
 We need a more abstract way to characterize these algorithms with respect to
 their execution times, one that is independent of the program or computer
-being used. This measure could be used to judge the algorithm alone and to
+being used. This method could then be used to analyze a single algorithm and
```

Comment:
Callback to the "method of analysis" we mentioned before; using the same words triggers memory recall better.

At least I tell myself that. :octopus: